tests/test_enable_ssl.py

```diff
@@ -0,0 +1,91 @@
+import pgautofailover_utils as pgautofailover
+from nose.tools import *
+
+import os
+
+cluster = None
+monitor = None
+node1 = None
+node2 = None
+
+def setup_module():
+    global cluster
+    cluster = pgautofailover.Cluster()
+
+def teardown_module():
+    cluster.destroy()
+
+
+def check_ssl_files(node):
+    for setting, f in [("ssl_key_file", "server.key"),
+                       ("ssl_cert_file", "server.crt")]:
+        file_path = os.path.join(node.datadir, f)
+        assert os.path.isfile(file_path)
+        eq_(node.pg_config_get(setting), file_path)
+
+
+def test_000_create_monitor():
+    global monitor
+    monitor = cluster.create_monitor("/tmp/enable/monitor")
+    monitor.run()
+    monitor.wait_until_pg_is_running()
+
+def test_001_init_primary():
+    global node1
+    node1 = cluster.create_datanode("/tmp/enable/node1")
+    node1.create()
+    node1.run()
+    assert node1.wait_until_state(target_state="single")
+
+def test_002_create_t1():
+    node1.run_sql_query("CREATE TABLE t1(a int)")
+    node1.run_sql_query("INSERT INTO t1 VALUES (1), (2)")
+
+def test_003_init_secondary():
+    global node2
+    node2 = cluster.create_datanode("/tmp/enable/node2")
+    node2.create()
+    node2.run()
+
+    assert node2.wait_until_state(target_state="secondary")
+    assert node1.wait_until_state(target_state="primary")
+
+def test_004_maintenance():
+    print()
+    print("Enabling maintenance on node2")
+    node2.enable_maintenance()
+    assert node2.wait_until_state(target_state="maintenance")
+
+def test_005_enable_ssl_monitor():
+    monitor.enable_ssl(sslSelfSigned=True, sslMode="require")
+    monitor.sleep(2) # we signaled, wait some time
+
+    assert monitor.config_get("ssl.sslmode") == "require"
```

Comment:
Let's do this check with `eq_` to, so it's easier to debug when they are ever wrong. (same for similar checks below)

Reply:
Done now.